server/security_integration_test.go

```diff
@@ -98,6 +99,28 @@ func (s *SecurityIntegrationTestSuite) Test_CreateAPIKey() {
 func (s *SecurityIntegrationTestSuite) Test_CreateAppPortalAPIKey() {
 	expectedStatusCode := http.StatusCreated
 
+	// Switch to the native realm
+	err := config.LoadConfig("./testdata/Auth_Config/full-convoy-with-native-auth-realm.json")
+	require.NoError(s.T(), err)
+
+	initRealmChain(s.T(), s.DB.APIRepo(), s.DB.UserRepo(), s.ConvoyApp.cache)
+
+	member, err := testdb.SeedOrganisationMember(s.DB, s.DefaultOrg, s.DefaultUser, &auth.Role{
+		Type:   auth.RoleAdmin,
+		Groups: []string{s.DefaultGroup.UID},
+	})
+
+	newAPIKey := &models.APIKey{
```

Comment:
@danvixent This is not an app portal API key. This test should fail. An app portal API key requires `appID`

Reply:
@Subomi It's seeding an API Key, for the request to use to authenticate to the API later on. It switches to the native realm, creates an API key for that group, and uses the key to authenticate & create an app portal key, the test is more comprehensive that way.

I will add comments to explain this properly.